.github/workflows/lint.yaml

```
@@ -0,0 +1,20 @@
+name: 'Lint codebase'
+
+on:
+  pull_request:
+    branches: [ "main", "dev" ]
```

```suggestion
    branches: [ "main" ]
```

I don't think we will have a dev branch. WDYT?